evennia/contrib/wilderness.py

```diff
@@ -0,0 +1,490 @@
+"""
+Wilderness system
+
+Evennia contrib - titeuf87 2017
+
+This contrib provides a wilderness map. This is an area that can be huge where
+the rooms are mostly similar, except for some small cosmetic changes like the
+room name.
+
+Usage:
+
+    This contrib does not provide any commands. Instead the @py command can be
+    used.
+
+    A wilderness map needs to created first. There can be different maps, all
```

Comment:
It would be useful with a very brief example of creating such a map, here or at the end of this docstring.